openassessment/fileupload/views_filesystem.py

```diff
@@ -21,7 +21,7 @@ def filesystem_storage(request, key):
     """
     Uploading and download files to the local filesystem backend.
     """
-    if isinstance(key, six.text_type):
+    if six.PY2 and isinstance(key, six.text_type):
```

Comment:
Why do we need to only do this in Py2, should this be safe in both py2 and py3?

Reply:
This is used for file handling. this is used to get the path and open files. open works for both bytes and unicode type path in both py2 and py3 but in py3 if we keep it bytes we will have to change on a lot of other place and make all other strings bytes which are hardcoded and unicode by default.

Reply:
In that case can we just always leave this string as unicode and not do this conversion?